src/services/user-group/logic/courses.js

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many blank lines at the beginning of file. Max of 0 allowed.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5369432)